marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/ModelController.java

```diff
@@ -92,8 +92,8 @@
 
     @RequestMapping(value = "/{modelName}/info", method = RequestMethod.PUT)
```

Comment:
@rjrudin 
Is it ok to leave this endpoint as  /{modelName}/info or should it be renamed ( /{modelName}/infoAndNamepace)?

Reply:
I think something is off here - "info" refers to model-level data, such as name/description (and possibly in the future, version and baseUri). 

namespace/namespacePrefix are definition-level data. So it doesn't make sense to save those together with model-level data. 

ES allows for each definition to have its own namespace/namespacePrefix. That certainly doesn't seem common, but it's allowable. So I think we need to allow that here as well - which means I think the story needs to be adjusted. 

If the story stays as-is, then the assumption would be that whatever the user specifies here should only be added to the "primary" entity definition. But - a user would likely still need a way to set this for the structured-type entity definitions as well.

Reply:
@rjrudin @srinathgit so on the UI side, if we want to enable the user to add a namespace and prefix to a structured property, that would go in this form:

<img width="613" alt="Screen Shot 2021-04-21 at 9 12 35 AM" src="https://user-images.githubusercontent.com/477757/115586997-03e94700-a282-11eb-8ec8-de703fb4b8dd.png">

For the UI work, if we are to do it, that should be a separate story. FYI: @sbayatpur